tests/serial/circular/Explicit.cpp

```diff
@@ -0,0 +1,21 @@
+#ifndef PRECICE_NO_MPI
+
+#include "helper.hpp"
+#include "testing/Testing.hpp"
+
+#include <precice/SolverInterface.hpp>
+
+BOOST_AUTO_TEST_SUITE(Integration)
+BOOST_AUTO_TEST_SUITE(Serial)
+BOOST_AUTO_TEST_SUITE(Circular)
```

Comment:
Why not adding this to the `three-solvers` namespace?

Reply:
I would like to add more circular tests with more participants.